sql/core/src/main/scala/org/apache/spark/sql/execution/command/resources.scala

```diff
@@ -100,3 +110,24 @@ case class ListJarsCommand(jars: Seq[String] = Seq.empty[String]) extends Runnab
     }
   }
 }
+
+/**
+ * Returns a list of archive paths that are added to resources.
+ * If file paths are provided, return the ones that are added to resources.
```

Comment:
`archive file paths` or `archive paths`?

Reply:
Thanks. I've missed to modify it.